extensions/wikia/Recirculation/js/utils.js

```diff
@@ -0,0 +1,40 @@
+/*global define*/
+define('ext.wikia.recirculation.utils', [
+	'wikia.loader',
+	'wikia.cache'
+], function (loader, cache) {
+	'use strict';
+
+	/**
+	 * Checks if template is cached in LocalStorage and if not loads it by using loader
+	 * @returns {$.Deferred}
+	 */
+	function loadTemplate(templateLocation) {
```

Comment:
Uhm, I think we could make something useful for all company here ;) How about moving this method, a little bit changed to `wikia.loader`? And then re-use it in `extensions/wikia/Recirculation/js/incontent.js` module this way:
```
if (abTest.inGroup('RECIRCULATION_INCONTENT', 'YES')) {
   loader
     .loadTemplate(
         'RecirculationAssets_',
         'extensions/wikia/Recirculation/templates/inContent.client.mustache'
     )
     .then(function(loadedTemplate) {
         template = loadedTemplate;
         injectInContentWidget($container);
     });
}
```
The method in `wikia.loader` will have this signature then:
```
function loadTemplate(cacheKey, templateLocation, cacheTtl)
```
where default `cacheTtl = 86400`.

And in future while touching some of [those pieces of code](https://github.com/Wikia/app/search?utf8=%E2%9C%93&q=function+loadTemplate%28) we could replace custom `loadTemplate()` calls there with `wikia.loader`'s `loadTemplate()` :gem: 

@macbre @lizlux @hakubo @RafLeszczynski and @kvas-damian what do you think about it?